polkadot/node/subsystem-util/src/runtime/mod.rs

```diff
@@ -469,61 +469,35 @@ where
 }
 
 /// Request the min backing votes value.
-/// Prior to runtime API version 6, just return a hardcoded constant.
 pub async fn request_min_backing_votes(
```

Comment:
we can remove these helpers altogether and implement them with `specialize_requests` like we do for the other runtime APIs